sklearn/tests/test_common.py

```diff
@@ -49,26 +51,52 @@ def test_all_estimators():
     # properly
     assert_greater(len(estimators), 0)
```

Comment:
Remove (or move to the appropriate location) the comment above "Test that estimators are default-constructible etc ..."